nixos/modules/virtualisation/digital-ocean-image.nix

```diff
@@ -0,0 +1,69 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.virtualisation.digitalOceanImage;
```

Comment:
Hi! @arcnmx, I am planning to use this custom image. But I am confused about where is this `digitalOceanImage` located? Because in the file `digital-ocean-image.nix`, I only found `digitalOcean`.

Reply:
You'll have to manually import the module file before you can use it.

This seems to be a convention all images use. Not sure if it's a good one. 

E.g.. https://github.com/nix-community/nixos-generators/blob/master/formats/gce.nix

Reply:
@arianvp thank you! Yes, I think I have misunderstanding about the use of `cfg = config.virtualisation.digitalOceanImage` here.